packages/storybook/stories/pattern-prefill/va-prefill-alert.stories.jsx

```diff
@@ -0,0 +1,248 @@
+import React from 'react';
+import { VaAlert } from '@department-of-veterans-affairs/web-components/react-bindings';
+import { getWebComponentDocs, propStructure, StoryDocs } from '../wc-helpers';
+
+const alertDocs = getWebComponentDocs('va-alert');
+// Remove backgroundOnly prop, this code and prop should be removed with the v1 version
+alertDocs.props = alertDocs.props.filter(
+  prop => prop.name !== 'backgroundOnly',
+);
+
+export default {
+  title: 'Patterns/Prefill/Components/Prefill Alert',
+  id: 'patterns/components/prefill-alert',
+  argTypes: {
+    headline: {
+      table: {
+        disable: true,
+      },
+    },
+    children: {
+      table: {
+        disable: true,
+      },
+    },
+  },
+  parameters: {
+    componentSubtitle: 'va prefill alert',
+    docs: {
+      page: () => (
+        <StoryDocs storyDefault={SignedInPrefillAlert} data={alertDocs} />
+      ),
+    },
+  },
+};
+
+// Fix for displaying component name when using bindings in 'Show code'
+VaAlert.displayName = 'VaAlert';
+
+const defaultArgs = {
+  'slim': false,
+  'status': 'info',
+  'disable-analytics': false,
+  'visible': true,
+  'close-btn-aria-label': 'Close notification',
+  'closeable': false,
+  'full-width': false,
+  //   'headline': (
+  //     <h2 id="track-your-status-on-mobile" slot="headline">
+  // Note: Since you’re signed in to your account, we can prefill part of your form based on your account details. You can also save your form in progress and come back later to finish filling it out.    </h2>
+  //   ),
+  'children': (
+    <p className="vads-u-margin-y--0">
+      <strong>Note:</strong> Since you’re signed in to your account, we can
+      prefill part of your form based on your account details. You can also save
+      your form in progress and come back later to finish filling it out.
+    </p>
+  ),
+};
+
+const Template = ({
+  slim,
+  status,
+  'disable-analytics': disableAnalytics,
+  visible,
+  'close-btn-aria-label': closeBtnAriaLabel,
+  closeable,
+  'full-width': fullWidth,
+  headline,
+  onCloseEvent,
+  children,
+}) => {
+  if (onCloseEvent)
+    return (
+      <VaAlert
+        slim={slim}
+        status={status}
+        disableAnalytics={disableAnalytics}
+        visible={visible}
+        closeBtnAriaLabel={closeBtnAriaLabel}
+        closeable={closeable}
+        fullWidth={fullWidth}
+        onCloseEvent={onCloseEvent}
```

Comment:
Do the event callbacks work in Storybook? If not, I wonder if we should just remove it from the example. The API for `va-alert` can be referenced for custom events.

Reply:
Yes, the event callbacks do work in Storybook. I do see your point about keeping the examples clean and focused. I decided to remove the unnecessary props that aren't relevant to our pattern from the alerts. Thanks for the suggestion!